hmac.go

```diff
@@ -180,7 +180,7 @@ func (h *opensslHMAC) Reset() {
 	}
 
 	runtime.KeepAlive(h) // Next line will keep h alive too; just making doubly sure.
-	h.sum = nil
+	h.sum = [maxHashSize]byte{}
```

Comment:
Same as in `evpHash.Sum`: better zero this slice when calling `opensslHMAC.Sum` rather than here.

Reply:
fixed